### PR TITLE
kv,cmd(ticdc): refinement about resolve lock

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

Issue Number: ref #4644

### What is changed and how it works?

* Add resolve lock cli
* Add resolve lock metrics
* Improve resolve lock logs

### Check List 

Tests 

 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
None
```

### CODE REVIEW

File: pkg/cmd/cli/cli_unsafe_resolve_lock.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 PingCAP, Inc.
```
Comment (hi-rustin) [inline pkg/cmd/cli/cli_unsafe_resolve_lock.go:1]:
```suggestion
// Copyright 2022 PingCAP, Inc.
```
😆

File: pkg/cmd/cli/cli_unsafe_resolve_lock.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		// Try not kill active transaction, we only resolves lock 1 minute ago.
+		o.ts = oracle.GoTimeToTS(now.Add(-time.Minute))
+	}
+
+	o.kvStorage, err = f.KvStorage()
+	return err
+}
+
+// run runs the `cli unsafe show-metadata` command.
+func (o *unsafeResolveLockOptions) run(_ *cobra.Command) error {
```
Comment (hi-rustin) [inline pkg/cmd/cli/cli_unsafe_resolve_lock.go:66]: Why not just delete this parameter?

File: pkg/cmd/cli/cli_unsafe_resolve_lock.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+// addFlags receives a *cobra.Command reference and binds
+// flags related to template printing to it.
+func (o *unsafeResolveLockOptions) addFlags(cmd *cobra.Command) {
+	if o == nil {
+		return
+	}
+
+	cmd.PersistentFlags().Uint64Var(&o.regionID, "region", 0, "Region ID")
```
Comment (hi-rustin) [inline pkg/cmd/cli/cli_unsafe_resolve_lock.go:86]:
```suggestion
	cmd.Flags().Uint64Var(&o.regionID, "region", 0, "Region ID")
```
There should be no subcommands that use this parameter?  Other commands actually misuse this interface, and I'll fix it when I have time.

**Review by hi-rustin — APPROVED** [review]
rest LGTM

File: pkg/cmd/factory/factory_impl.go [hunk 4]
```diff
@@ -19,6 +19,8 @@ import (
 	"time"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/kv"
+	cdckv "github.com/pingcap/tiflow/cdc/kv"
```
Comment (3AceShowHand) [inline pkg/cmd/factory/factory_impl.go:23]: in some file, `/tidb/kv` was renamed to `tidbkv`.
I would prefer to follow that convention. `kv` for `/cdc/kv`

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**